client/my-sites/upgrades/domain-management/email-forwarding/email-forwarding-add-new.jsx

```diff
@@ -20,6 +20,7 @@ import analyticsMixin from 'lib/mixins/analytics';
 import notices from 'notices';
 import * as upgradesActions from 'lib/upgrades/actions';
 import { validateAllFields } from 'lib/domains/email-forwarding';
+import supportPaths from 'lib/url/support';
```

Comment:
Looking at other places where this is used, this import is usually named `support`. Or `supportUrls`, which is more fitting give the path `lib/*url*/support` ;) Same comment for other files.